client/checks_test.go

```diff
@@ -109,3 +109,39 @@ func (cs *clientSuite) TestStopChecks(c *check.C) {
 	c.Check(body["action"], check.Equals, "stop")
 	c.Check(body["checks"], check.DeepEquals, []any{"chk1", "chk2"})
 }
+
+func (cs *clientSuite) TestRefreshCheck(c *check.C) {
+	cs.rsp = `{
+		"result": {
+			"info": {
+				"name": "chk1",
+				"startup": "enabled",
+				"status": "up",
+				"failures": 0,
+				"threshold": 3,
+				"change-id": "42"
+			},
+			"error": ""
```

Comment:
Per my suggestion below, if we make this field `omitempty` on the server, let's remove it here.

Also, let's add another test with an actual error message.